doc/api/events.md

```diff
@@ -1834,10 +1834,12 @@ setMaxListeners(5, target, emitter);
 added:
  - v20.5.0
  - v18.18.0
+changes:
+ - version: REPLACEME
+   pr-url: https://github.com/nodejs/node/pull/57765
+   description: API is now stable.
```

Comment:
FWIW we usually use `is no longer experimental`, we try to avoid `now` as it could be confusing to future reader if the status of the API ever changes. Let's roll back to the usual "is no longer experimental", or we can go with what Rich suggested